src/main/java/mate/academy/service/BookServiceImpl.java

```diff
@@ -42,10 +42,12 @@ public void deleteById(Long id) {
     }
 
     @Override
-    public void updateById(Long id, CreateBookRequestDto bookWithoutId) {
+    public BookDto updateById(Long id, CreateBookRequestDto bookWithoutId) {
         Book book = bookMapper.toModel(bookWithoutId);
```

Comment:
It's better to 
```suggestion
        Book book = bookRepository.findById(id)
                .orElseThrow(() -> new EntityNotFoundException(getNotFoundMessage(id)));
        bookMapper.updateBookFromDto(bookDto, book);
```
where in BookMapper
`void updateBookFromDto(CreateBookRequestDto book, @MappingTarget Book entity);`

Reply:
Should we set categories here? 
```
book.setCategories(categoriesIdToCategories(bookDto.categoriesId()));
return bookMapper.toDto(bookRepository.save(book));
```